Documentation/contributing/development/contributing_guide.rst

```diff
@@ -235,43 +320,73 @@ Pull request review process
    note suitability. Put yourself into the perspective of a future release
    notes reader with lack of context and ensure the title is precise but brief.
 
-   +-----------------------------------+---------------------------------------------------------------------------+
-   | Labels                            | When to set                                                               |
-   +===================================+===========================================================================+
-   | ``dont-merge/needs-release-note`` | Do NOT merge PR, needs a release note                                     |
-   +-----------------------------------+---------------------------------------------------------------------------+
-   | ``release-note/bug``              | This is a non-trivial bugfix                                              |
-   +-----------------------------------+---------------------------------------------------------------------------+
-   | ``release-note/major``            | This is a major feature addition                                          |
-   +-----------------------------------+---------------------------------------------------------------------------+
-   | ``release-note/minor``            | This is a minor feature addition                                          |
-   +-----------------------------------+---------------------------------------------------------------------------+
+   +-----------------------------------+--------------------------------------------------------------------------------------------+
+   | Labels                            | When to set                                                                                |
+   +===================================+============================================================================================+
+   | ``dont-merge/needs-release-note`` | Do NOT merge PR, needs a release note                                                      |
+   +-----------------------------------+--------------------------------------------------------------------------------------------+
+   | ``release-note/bug``              | This is a non-trivial bugfix and is a user facing bug                                      |
+   +-----------------------------------+--------------------------------------------------------------------------------------------+
+   | ``release-note/major``            | This is a major feature addition, e.g. Add MongoDB support                                 |
+   +-----------------------------------+--------------------------------------------------------------------------------------------+
+   | ``release-note/minor``            | This is a minor feature addition, e.g. Add support for a Kubernetes version                |
```

Comment:
Question: How do you see performance improvements?

I'm thinking even micro-optimizations like https://github.com/cilium/cilium/pull/11089 . It'd be nice to highlight that in the blog, but I don't know if it's reasonable to add such an item to the `minor` section.

Also, I think it'd be useful to highlight that `minor` should be user-facing. `misc` should not be user-facing. If you are adding `misc` and then realise there is some user impact of any kind, it should be `minor` instead. I think that the descriptions in the github labels try to capture this.